main/main.cpp

```diff
@@ -1489,8 +1489,8 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	GLOBAL_DEF("input_devices/pointing/ios/touch_delay", 0.150);
 
 	// XR project settings.
-	GLOBAL_DEF_BASIC("xr/openxr/enabled", false);
-	GLOBAL_DEF_BASIC("xr/openxr/default_action_map", "res://default_action_map.tres");
+	GLOBAL_DEF_RST("xr/openxr/enabled", false);
```

Comment:
Shouldn't this use `GLOBAL_DEF_RST_BASIC`? Otherwise this setting will be advanced i.e. hidden by default, while all the OpenXR config settings are marked as basic, i.e. always visible.

Reply:
oops yet